content/source/docs/cloud/sentinel/import/tfrun.html.md

```diff
@@ -42,6 +47,58 @@ to only enforce the policy on non-development workspaces is more appropriate.
 [run-glossary]: /docs/glossary.html#run
 [workspace-glossary]: /docs/glossary.html#workspace
 
+## Namespace: root
+
+The **root namespace** contains data associated with the current run.
+
+### Value: `created_at`
+
+* **Value Type:** String.
+
+The `created_at` value within the [root namespace](#root-namespace) specifies the time that the run was created.
```

Comment:
If this is a string I assume it's following some particular datetime format, could we be explicit about what that is and maybe provide an example to avoid any confusion?

Reply:
@glenngillen it is an rfc3339 compliant format. I'll call this out in the docs and I will provide an example of how we can use the time import to load the value and act on it

Reply:
@glenngillen pushed some changes, please review and resolve conversation if happy with changes. Thanks.